presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveSchema.java

```diff
@@ -0,0 +1,285 @@
+                .map(QueryAssert.Row::row)
+                .collect(toImmutableList());
+
+        // SHOW SCHEMAS
+        assertThat(onPresto().executeQuery("SHOW SCHEMAS FROM hive"))
+                .satisfies(containsFirstColumnValue("information_schema"));
+
+        // SHOW TABLES
+        assertThat(onPresto().executeQuery("SHOW TABLES FROM hive.information_schema"))
+                .satisfies(containsFirstColumnValue("tables"))
+                .satisfies(containsFirstColumnValue("columns"))
+                .satisfies(containsFirstColumnValue("table_privileges"))
+                .doesNotHave(containsFirstColumnValue("column_privileges")); // Hive 3's information_schema has column_privileges view
+
+        // SHOW COLUMNS
+        assertThat(onPresto().executeQuery("SHOW COLUMNS FROM hive.information_schema.columns"))
+                .satisfies(containsFirstColumnValue("table_catalog"))
+                .satisfies(containsFirstColumnValue("table_schema"))
+                .doesNotHave(containsFirstColumnValue("is_updatable")); // Hive 3's information_schema.columns has is_updatable column
+
+        assertThat(() -> onPresto().executeQuery("SHOW COLUMNS FROM hive.information_schema.column_privileges")) // Hive 3's information_schema has column_privileges view
+                .failsWithMessage("line 1:1: Table 'hive.information_schema.column_privileges' does not exist");
+
+        // DESCRIBE
+        assertThat(onPresto().executeQuery("DESCRIBE hive.information_schema.columns"))
+                .satisfies(containsFirstColumnValue("table_catalog"))
+                .satisfies(containsFirstColumnValue("table_schema"))
+                .satisfies(containsFirstColumnValue("column_name"))
+                .doesNotHave(containsFirstColumnValue("is_updatable")); // Hive 3's information_schema.columns has is_updatable column
+
+        assertThat(() -> onPresto().executeQuery("DESCRIBE hive.information_schema.column_privileges")) // Hive 3's information_schema has column_privileges view
+                .failsWithMessage("line 1:1: Table 'hive.information_schema.column_privileges' does not exist");
+
+        // information_schema.schemata
+        assertThat(onPresto().executeQuery("SELECT schema_name FROM information_schema.schemata"))
+                .satisfies(containsFirstColumnValue("information_schema"));
+
+        // information_schema.tables
+        assertThat(onPresto().executeQuery("SELECT DISTINCT table_schema FROM information_schema.tables"))
+                .satisfies(containsFirstColumnValue("information_schema"));
+        assertThat(onPresto().executeQuery("SELECT table_name FROM information_schema.tables WHERE table_schema = 'information_schema'"))
+                .containsOnly(allInformationSchemaTablesAsRows);
+        Assertions.assertThat(onPresto().executeQuery("SELECT table_schema, table_name FROM information_schema.tables").rows().stream()
```

Comment:
static import

Reply:
> static import

`assertThat`?
I use `tempto QueryAssert.assertThat` and `assertj Assertions.assertThat` -- it happens; not pretty; we already do this in other tests